Lib/pathlib.py

```diff
@@ -479,6 +260,113 @@ def __reduce__(self):
         # when pickling related paths.
         return (self.__class__, tuple(self._parts))
 
+    @classmethod
+    def _split_extended_path(cls, s):
+        prefix = ''
+        if s.startswith(_WIN_EXT_NAMESPACE_PREFIX):
+            prefix = s[:4]
+            s = s[4:]
```

Comment:
Is `_WIN_EXT_NAMESPACE_PREFIX` 4 characters long and that's why you are slicing that much? If so:
```suggestion
            prefix = _WIN_EXT_NAMESPACE_PREFIX
            s = s.removesuffix(_WIN_EXT_NAMESPACE_PREFIX)
```

Regardless, the length of 3 seems a bit magical.

Reply:
I'd like to leave this code alone (aside from moving it) as it'll be removed in #91882 - see my top-level comment. Hope that makes sense! Resolving this thread...